scripts/components/advanced-color-picker/docs/story.js

```diff
@@ -1,15 +1,9 @@
 import React, { useState } from 'react';
-import readme from './readme.mdx';
 import { AdvancedColorPicker } from '../advanced-color-picker';
 import { SingleItemShowcase } from '../../../storybook/helpers';
 
 export default {
-	title: `Options/AdvancedColorPicker`,
-	parameters: {
-		docs: {
-			page: readme
-		}
-	},
+	title: 'Options/AdvancedColorPicker',
```

Comment:
```suggestion
	title: 'Options/Advanced Color Picker',
```

Reply:
I wanted to keep these as-is, so they match the actual component name